docs_input/api/linalg/matvec/matmul.rst

```diff
@@ -13,6 +13,8 @@ is supported for any tensor with a rank higher than 2.
 .. doxygenfunction:: matmul(const OpA &A, const OpB &B, float alpha = 1.0, float beta = 0.0)
 .. doxygenfunction:: matmul(const OpA &A, const OpB &B, const int32_t (&axis)[2], float alpha = 1.0, float beta = 0.0)
 
+For information on experimental sparse tensor support for SpMM, please see :ref:`_sparse_tensor_api`.
```

Comment:
maybe write "for sparse * sparse (SpMM)" for people not familiar with the term

Reply:
done